### PR TITLE
feat(paging): implemented paging for cira, domains, devices, profiles and wifi

### PR DESCRIPTION
**Pull request by Mahendra-intel**

### CODE REVIEW

File: src/app/devices/devices.service.ts [hunk 1]
```diff
@@ -214,12 +214,12 @@ export class DevicesService {
       )
   }
 
-  getDevices (tags: string[] = []): Observable<Device[]> {
+  getDevices (pageSize: any, startsFrom: any, count: any, tags: string[] = []): Observable<DeviceResponse> {
```
Comment (rsdmike) [inline src/app/devices/devices.service.ts:217]: we can type these

File: src/app/configs/configs.component.ts [hunk 2]
```diff
@@ -65,6 +73,15 @@ export class ConfigsComponent implements OnInit {
     })
   }
 
+  pageChanged (event: any): void {
```
Comment (rsdmike) [inline src/app/configs/configs.component.ts:76]: can we type this?

**Review by rsdmike — CHANGES_REQUESTED** [review]
Looking good, just need to minimize the anys

File: src/app/devices/devices.service.ts (hunk 1 above)
Comment (Mahendra-intel) [reply src/app/devices/devices.service.ts]: resolved

File: src/app/configs/configs.component.ts (hunk 2 above)
Comment (Mahendra-intel) [reply src/app/configs/configs.component.ts]: resolved

**Review by rsdmike — APPROVED** [review]

**Pull request merged**